es-ds-docs/pages/molecules/button.vue

```diff
@@ -0,0 +1,532 @@
+<template>
+    <div>
+        <h1>
+            Button
+        </h1>
+        <p>
+            Extended from <a
+                href="https://v3.primevue.org/button/"
+                target="_blank">
+                primevue button
+            </a>
+        </p>
+
+        <p>
+            When using a button, please ensure that the copy style is
+            <b-link
```

Comment:
References to b-link need to be removed and replaced with either `a` or `nuxt-link`, because BootstrapVue is no longer present. At the moment there's hydration errors as a result of the missing component